### ISSUE TITLE
Can't use all available methods to deal with weak subjectivity

### ISSUE THREAD

**Issue by thomkeh** [eth-educators/eth-docker#479]
If I understand [the Teku docs](https://docs.teku.consensys.net/en/latest/Concepts/Weak-Subjectivity/) correctly, it is unsafe to sync the beaconchain if you don't have a relatively recent checkpoint to guide you because of long-range attacks.

Teku has two methods to deal with that:

1. Use `--initial-state` to supply an SSZ encoded state file from which to sync.
2. Use `--ws-checkpoint` to supply a weak subjectivity checkpoint (which is basically just a hash) by which a node can securely validate its view of the current state.

[Lighthouse seems to only have](https://lighthouse-book.sigmaprime.io/checkpoint-sync.html) the _first_ method with the flag `--checkpoint-sync-url` whereas [Prysm seems to only have](https://docs.prylabs.network/docs/prysm-usage/parameters/) the _second_ method with the flag `--weak-subjectivity-checkpoint`. (I don't know the situation with the other clients.)

eth-docker supports using the first method, but it currently doesn't have support for the second method.

**Comment by yorickdowne** [comment]
Correct, eth-docker out of the box only supports initial state / checkpoint sync, on the clients which support this: Which is all but Prysm.

A weak subjectivity checkpoint on its own doesn't speed up sync and has no advantage outside of using an archive node.

That said, an enterprising user can absolutely use the existing support for Lighthouse, Teku or Lodestar to do a `--ws-checkpoint` (or equivalent) instead, simply by setting the RAPID_SYNC variable accordingly: It is just added as a parameter without further checking.
